tailwind.config.js

```diff
@@ -25,19 +25,25 @@ module.exports = {
       },
     },
     screens: {
+      // eslint-disable-next-line prettier/prettier
       'xs': '468px',
+      // eslint-disable-next-line prettier/prettier
       'sm': '640px',
+      // eslint-disable-next-line prettier/prettier
       'md': '768px',
+      // eslint-disable-next-line prettier/prettier
       'lg': '1024px',
+      // eslint-disable-next-line prettier/prettier
       'xl': '1280px',
+      // eslint-disable-next-line prettier/prettier
       'xxl': '1536px',
```

Comment:
What is this error showing exactly 🤔 

And can we use something like below if its needed? 
```suggestion
      // eslint-disable prettier/prettier
      'xs': '468px',
      'sm': '640px',
      'md': '768px',
      'lg': '1024px',
      'xl': '1280px',
      'xxl': '1536px',
      // eslint-enable prettier/prettier
```

Reply:
Documentation lied to me seems like we can do it like this instead

![image](https://user-images.githubusercontent.com/28055566/174491756-134c8c49-0121-4526-bd33-d63d05990293.png)